http/services/authService.js

```diff
@@ -0,0 +1,17 @@
+import HttpService from "./httpService";
+
+class AuthService extends HttpService {
+    constructor() {
+        super();
+        this.login = this.login.bind(this);
+    }
+
+    async login(payload) {
+        const data = await this.client.post("/auth/login", payload);
```

Comment:
napravi konstantu iznad servisa koja ce se zvati ENDPOINTS i tu definisi sve endpointe koje koristis

mozes ovde const {accessToken} = await this...